src/index.html

```diff
@@ -15,8 +15,104 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body class="first">
+    <header>
+      <a
+        href="#moyo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a
+              href="#apple"
+              class="navigation-link is-active"
+            >
+              APPLE
+            </a>
```

Comment:
A new line is needed between sibling blocks of HTML. It helps to add some 'air' and simplify reading. Please, add a new line between each `li` elements.